tendrl/commons/utils/cmd_utils.py

```diff
@@ -9,7 +9,6 @@
 
 SAFE_COMMAND_LIST = [
     "lsblk",
-    "cat",
```

Comment:
Check all our source and make sure we are not using 'cat' anywhere?
Please confirm